.github/workflows/semver-label.yml

```
@@ -16,15 +16,13 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4 # v6
 
-      - name: Install pnpm
+      - name: Setup Node
         uses: pnpm/action-setup@v4
         with:
-          version: "9.12.0"
-
-      - name: Setup Node
+          version: 9.12.0
         uses: actions/setup-node@v4 # v6
```

YAMLlint (1.38.0)

[error] duplication of key "uses" in mapping

(key-duplicates)